packages/teko/tests/src/Tpetra/tBlockedTpetraOperator.cpp

```diff
@@ -180,19 +179,18 @@ bool tBlockedTpetraOperator::test_vector_constr(int verbosity,std::ostream & os)
    TEST_MSG("\n   tBlockedTpetraOperator::test_vector_constr: "
          << "Running on " << comm_epetra.NumProc() << " processors");
 
-   // pick 
+   // pick
    int nx = 5;// * comm_epetra.NumProc();
    int ny = 5;// * comm_epetra.NumProc();
 
    // create a big matrix to play with
    // note: this matrix is not really strided
    //       however, I just need a nontrivial
    //       matrix to play with
-   Trilinos_Util::CrsMatrixGallery FGallery("recirc_2d",comm_epetra,false);
+   CrsMatrixGallery FGallery("recirc_2d");
```

Comment:
Why is the communicator not needed anymore? Is this now always using COMM_WORLD?